### ISSUE TITLE
Auto switch theme (at night/daytime)

### ISSUE THREAD

**Issue by rugk** [adrianolaru/switcheroo#4]
It would be great if you could add some features to auto-switch a theme.
1. the first approach is quite simple: a time-based auto switch it switches between dark/light
2. as mentioned by @adrianolaru in #1 by detecting luminosity with the camera
3. However as I assume this is a bit difficult you could also just use a mix of 1 and 2 by locating the user (or let him enter geo coordinates) and calculate the sunrise and sunset times where you automatically switch. This way it's e.g. done by satnavs or software like [f.lux] (https://justgetflux.com/).

Basically I would really like the first thing to be implemented and maybe the second and/or third thing as an 'extra-feature'. :smiley:

**Comment by adrianolaru** [comment]
Hey @rugk, first I need to make this work for atom 1.0, and then will see... I've started using atom again, so I will make time to fix it first...

**Comment by rugk** [comment]
Okay, great. Of course making it working is more important - but afterwards you can have a look at this idea again.
